flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/SqlITCase.scala

```diff
@@ -317,4 +317,92 @@ class SqlITCase extends StreamingWithStateTestBase {
     result.addSink(new StreamITCase.StringSink)
     env.execute()
   }
+  
+  
+  @Test
+  def testAvgSumAggregatationPartition(): Unit = {
```

Comment:
I looked over again. If you try to import
import org.apache.flink.streaming.util. ...you do not have the options to use KeyedOneInputStreamOperatorTestHarness/OneInputStreamOperatorTestHarness

If it is fine for you i would prefer to keeps things simple and either use the custom source (as it is not very heavy) or a custom time assignment. Finally...these works and does the job to test

@fhueske